content/post/twitter-hashtags-real-time.md

```diff
@@ -0,0 +1,561 @@
+                            'rgba(75, 192, 192, 1)',
+                            'rgba(153, 102, 255, 1)'
+                        ],
+                        borderWidth: 1
+                    }]
+                },
+                options: {
+                    scales: {
+                        yAxes: [{
+                            ticks: {
+                                beginAtZero:true
+                            }
+                        }]
+                    }
+                }
+           });
+
+
+           var src_Labels = [];
+           var src_Data = [];
+
+            setInterval(function(){
+                $.getJSON('/refreshDashboard', {
+                }, function(data) {
+                    src_Labels = data.r_hashtags;
+                    src_Data = data.r_counts;
+                });
+
+                myChart.data.labels = src_Labels;
+                myChart.data.datasets[0].data = src_Data;
+                myChart.update();
+
+            },1000);
+
+</script>
+```
+
+### 13. Run The Application
+
+Before we run our Twitter Trending Hashtags application, we need to make sure we have the proper Python dependencies installed. We depend on `pandas`, `requests_oauthlib`, and `flask`. These can be installed with the following commands:
+
+```
+pip install pandas requests_oauthlib flask
+```
+
+Now that we have built all the components, from grabbing the data all the way to representing it on a dashboard, the only remaining step it to run everything:
+
+1. Run the Dashboard application: **app.py**
+2. Run **socket\_receiver.py**
+3. Run the **twitter\_wallaroo\_app** from terminal using the below commands:
+
+Note: The `machida` executable is in `machida/build/machida` in the [Wallaroo](https://github.com/wallaroolabs/wallaroo) repo. For example, if you’ve followed the Wallaroo installation [instructions](https://docs.wallaroolabs.com/book/getting-started/setup.html) then it will be in `$HOME/wallaroo-tutorial/wallaroo/machida/build/machida`. In order to run `machida` you will need to set up your `PYTHONPATH` to point to the `wallaroo.py` python library. For example, if you’ve followed the [Wallaroo installation instructions](https://docs.wallaroolabs.com/book/getting-started/setup.html) then `machida` will be `$HOME/wallaroo-tutorial/wallaroo/machida/build/machida` and you can set `PYTHONPATH` with `export PYTHONPATH=$HOME/wallaroo-tutorial/wallaroo/machida`
+
+```
+export PYTHONPATH="$PYTHONPATH:$HOME/wallaroo-tutorial/wallaroo/machida:."
+```
+
+```
+$HOME/wallaroo-tutorial/wallaroo/machida/build/machida --application-module twitter_wallaroo_app \
+  --in 127.0.0.1:8002 \
+  --out 127.0.0.1:7002 \
+  --metrics 127.0.0.1:5001 \
+  --control 127.0.0.1:6000 \
+  --data 127.0.0.1:6001 \
+  --worker-name worker1 \
+  --ponythreads=1 \
+  --external 127.0.0.1:5050 \
+  --cluster-initializer \
+  --ponythreads=1 \
+  --ponynoblock
+```
+
+4.  Run **twitter_client.py**
```

Comment:
Since the indentation above is broken, this appears as a new numbered bullet point (starting from 1)